data/defscript/classes.kvs

```diff
@@ -15,19 +15,19 @@ class(installer,object)
 	{
 		if($0 == "")
 		{
-			echo $tr("[Installer] Error: the first argument of copyFiles must be a source directory","defscript")
+			echo $tr("[Installer] Error: The first argument of copyFiles must be a source directory","defscript")
```

Comment:
The colon isnt acting as a sentence expansion but as a delimiter, the sentence starts after the colon itself.
```
Incorrect: I want: butter, sugar, and flour.

Correct:
I want the following: butter, sugar, and flour.
OR
I want butter, sugar, and flour.

Incorrect: I've seen the greats, including: Barrymore, Guinness, and Streep.

Correct: I've seen the greats, including Barrymore, Guinness, and Streep.
```